charts/nx-cloud/templates/nx-cloud-message-queue-deployment.yaml

```diff
@@ -19,8 +19,8 @@ spec:
     spec:
       containers:
         - name: nx-cloud-messagequeue
-          image: nxprivatecloud/nx-cloud-messagequeue:latest
-          imagePullPolicy: Always
+          image: {{ include "nxCloud.images.messagequeue.image" . }}
+          imagePullPolicy: {{ .Values.messagequeue.image.pullPolicy | quote }}
```

Comment:
we have the `charts/nx-cloud/values.yaml` file - where we put defaults

with your recent changes, people will always need to use that file, no? Otherwise they'll get error that the pullPolicy for the messagequeue is undefined etc.

if so, should we update the docs [here](https://github.com/nrwl/nx-cloud-helm#step-3-install-nx-cloud-using-helm)?

Reply:
messagequeue is one of the only items which has a default image + tag (`latest`) due to how it is currently published and that is defined in the base [`values.yaml`](https://github.com/nrwl/nx-cloud-helm/blob/788a069d066dc33c451e5421fae9ae470c9a5abb/charts/nx-cloud/values.yaml#L98-L100) file

Reply:
A values.yaml file is something users have always needed (or via --set on cli which doesn't scale), as that sets other critical parts of the configuration for rendering the templates